### DISCUSSION TITLE
Linux distro considerations

### DISCUSSION THREAD

**Discussion by elraval** [g4ixt/QtTinySA#15] [Q&A] [answered]
Hi. 

Just recently found about this software in tinySA Groups.io forum. I'm eager to check it out, but I understand this is Linux only, right?

I have a spare SSD in my computer that I can use to dual-boot Linux on. I've been out of Linux for 5 years or so now, and I don't know what distro would be best for me to try this out. Would you care to recomend a distro, please? 

Consider that I would probabily would also want to run GNURadio and other SDR software on it. I have an HackRF and a SDRPlay RSP1A.

Thank you in advance.

D

**Comment by g4ixt** [comment, marked as answer]
Hi D,

It runs in Python, and although I only use Linux I am told it also runs ok in Python on Windows and Mac.  I use Ubuntu 22.04 LTS  and Kubuntu 22.04 LTS (and also run GNURadio and GQRX).  It runs fine on a Raspberry Pi 4b if you have one of those.

I am trying to build a Linux executable but it comes out very large.  If I can get that to work I will try to build a Windows executable as well but that may be a long way off.

Regards

Ian
